internal/distroregistry/distroregistry.go

```diff
@@ -126,3 +129,23 @@ func mangleHostDistroName(name string, isBeta, isStream bool) string {
 func (r *Registry) FromHost() distro.Distro {
 	return r.hostDistro
 }
+
+// FromHost returns a distro instance, that is specific to the host.
+// Its name may differ from other supported distros, if the host version
+// is e.g. a Beta or a Stream.
+func (r *Registry) FallbackHost() (distro.Distro, string, map[string][]rpmmd.RepoConfig) {
+	hostDistroName, _, _, _ := distro.GetHostDistroName()
+
+	parts := strings.Split(hostDistroName, "-")
+	if len(parts) != 2 {
+		panic(hostDistroName + " is not valid distro name")
+	}
+	distroVersion := parts[1]
+
+	if strings.HasPrefix(hostDistroName, "fedora-") {
+		log.Println("Warning: " + hostDistroName + " is not supported Fedora version. " +
+			"osbuild-composer will start with Fedora 33 image definitions but this might not work!")
+		r.hostDistro = fedora33.NewUnreleasedDistro(hostDistroName, distroVersion)
+	}
+	return r.hostDistro, hostDistroName, unsupported.GenerateFedoraRepositories(distroVersion)
+}
```

Comment:
I would suggest to move this logic into `FromHost()` and not to export this as an additional method to prevent the need to implement additional logic by the caller to actually get the Host `distro.Distro`.